client/stylesheets/_toCat.scss

```diff
@@ -298,6 +298,14 @@ code {
     text-underline: underline;
 }
 
+.profileBtn {
+    background-color: $color-venetian-red !important;
+    border-color: $color-venetian-red !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces